AsyncDisplayKit/ASControlNode.mm

```diff
@@ -278,32 +275,28 @@ - (void)addTarget:(id)target action:(SEL)action forControlEvents:(ASControlNodeE
   _ASEnumerateControlEventsIncludedInMaskWithBlock(controlEventMask, ^
     (ASControlNodeEvent controlEvent)
     {
+      // Create new target action pair
+      ASControlTargetAction *targetAction = [[ASControlTargetAction alloc] init];
```

Comment:
Let's save resources, by sharing this targetAction for all events in the mask, by creating the object outside of the enumeration.